gradle/repositories/build.gradle

```diff
@@ -1,6 +1,7 @@
 repositories {
     mavenCentral()
     jcenter()
+    mavenLocal()
```

Comment:
This shouldn't have changed

Reply:
This will be removed when web3j is released.